etc/config/c++.amazon.properties

```diff
@@ -1355,12 +1355,18 @@ compiler.arm64gm101a2.semver=10.1.2
 
 ###############################
 # GCC for Kalray
-group.kalray.compilers=kvxg1031_v4111:kvxg1031_v4100:kvxg941_v490:kvxg941_v480:kvxg941_v460:kvxg750_v440:kvxg750_v430:kvxg750_v420:kvxg750_v410:kvxg750:k1cg741:k1cg750
+group.kalray.compilers=kvxg1130_v4120:kvxg1031_v4111:kvxg1031_v4100:kvxg941_v490:kvxg941_v480:kvxg941_v460:kvxg750_v440:kvxg750_v430:kvxg750_v420:kvxg750_v410:kvxg750:k1cg741:k1cg750
 group.kalray.groupName=Kalray MPPA GCC
 group.kalray.isSemVer=true
 
 # kvx versions are different from the GCC they wrap
 
+compiler.kvxg1130_v4120.exe=/opt/compiler-explorer/kvx/acb-4.12.0/bin/kvx-elf-g++
+compiler.kvxg1130_v4120.name=KVX ACB 4.12.0 (GCC 11.3.0)
+compiler.kvxg1130_v4120.semver=4.12.0
+compiler.kvxg1130_v4120.objdumper=/opt/compiler-explorer/kvx/acb-4.12.0/bin/kvx-elf-objdump
+compiler.kvxg1130_v4120.alias=kvxg1130_v4120
```

Comment:
```suggestion

```

The alias is not needed/wanted. I've used it because previous compiler was not working but we already had published it. We could have simply fixed the compiler... but decided to fix it this way. TLDR: you don't need the alias.